openstack-client-connectors/resteasy-connector/pom.xml

```diff
@@ -15,9 +15,9 @@
   		<version>2.3.2.Final</version>
   	</dependency>
   	<dependency>
-  		<groupId>org.codehaus.jackson</groupId>
-  		<artifactId>jackson-jaxrs</artifactId>
-  		<version>1.9.13</version>
+		<groupId>com.fasterxml.jackson.jaxrs</groupId>
+		<artifactId>jackson-jaxrs-json-provider</artifactId>
+		<version>2.9.8</version>
```

Comment:
I think this dependency may be removed because the same exists in parent pom

Reply:
I removed the dependency from the parent.